src/gameboy/include/gameboy/gameboy.h

```diff
@@ -69,6 +70,11 @@ class Gameboy {
 	 */
 	std::unique_ptr<GPU> gpu;
 
+	/**
+	 * Indicate whether the Recording Controller is running or not
+	 */
+	bool is_recording = false;
```

Comment:
This doesn't need to be a member here, let it simply be a flag that is passed to the constructor. After construction, we don't require this flag.

Reply:
Agreed.